File: C_Programming/Practice/Logic_Building/program4.c
// Addition of 2 numbers
#include<stdio.h>

int main()
{
    int i,j,k;  // local variable

    printf("Enter First Number : \n");
    scanf("%d",&i);

    printf("Enter Second Number : \n");
    scanf("%d",&j);

    k = i + j;   // business logic 

    printf("Addition is : %d\n",k);

    return 0;
}

File: Python/Competitive/Assignment/program4.py
Value = int(input("Enter number : "))

def CheckDivisible(No):
    if ((Value % 3 == 0) and (Value % 5 == 0)):
        print("Number is divisible by 3 & 5")
    else:
        print("Number is not divisible by 3 & 5")
        
CheckDivisible(Value)